fileServer.js

```diff
@@ -17,96 +19,69 @@
 
  function send404Request(res) {
    fs.readFile('public/404.html', 'UTF-8', function(err, html) {
-<<<<<<< HEAD
      if (err) {
        res.writeHead(500);
-       res.write("Internal Server Error");
+       res.write('Internal Server Error');
+       console.log(err);
      } else {
        res.writeHead(404, {
-         'Content-type': mimeType
+         'Content-type': 'text/html'
        });
        res.end(html);
      }
-=======
-     res.setHeader('Content-type', mimeType);
-     res.end(html);
->>>>>>> 945100f8f04e33242f280c0f03422c5c690fe44b
    });
  }
 
  function onRequest(req, res) {
-<<<<<<< HEAD
-   console.log('request starting....');
-=======
-   console.log('request starting...');
->>>>>>> 945100f8f04e33242f280c0f03422c5c690fe44b
-
-   var filename;
-   var pathname = url.parse(req.url).pathname;
-   filename = path.join(rootDir, pathname);
-   console.log('Filename: ' + filename);
-<<<<<<< HEAD
+   console.log(`${req.method} ${req.url}`);
+
+   var pathname = url.parse(req.url).path;
+
+   
+   var filename = path.join(rootDir, pathname);
+
+   var headers = {'Content-type': String(mimeType[path.extname 
```

Comment:
Make the code that figures out a mime type based on a file extension into a function so it is easy to get the mime Type for a file anywhere in this code!